Sign in with Concur for Enterprise Applications

brittnwestSAP opened the pull request:
Describing enterprise application integrations in more detail. Providing guidelines for email verification for companies using email address as a unique identifier (not unique in Concur so requires verification before first sign in).

brittnwestSAP committed:
Updating session maintenance and ent app guideline

Session should be maintained or ended using token. Added guidelines for enterprise applications.

brittnwestSAP committed:
Adding SIWC enterprise diagrams

brittnwestSAP committed:
Adding OTP reference and diagrams

Added reference to OTP documentation where relevant for enterprise applications and diagrams of the process.

brittnwestSAP committed:
Clarifying roles & permissions requirements

charlieconcur reviewed (changes requested):
Do not use underscores (`_`) in file names -- use a dash instead (`-`).

brittnwestSAP committed:
updating file names

brittnwestSAP committed:
removing file with underscores in name

brittnwestSAP committed:
removing file with underscores in name

brittnwestSAP committed:
Updating file names to use dash

brittnwestSAP committed:
adding dash to file names

brittnwestSAP committed:
new folder

brittnwestSAP committed:
Delete sign-in-with-concur-images

brittnwestSAP committed:
Creating new folder

brittnwestSAP committed:
Loading files in to new folder

brittnwestSAP committed:
Updating underscores to dashes

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Removing files in underscore folder

brittnwestSAP committed:
Adding prerelease html

brittnwestSAP commented:
updated new files and migrated old to dashes rather than underscore. Also changed full path to use dash - this is currently in pre-release and only affects 2 partners (whom I will notify specifically of the path change). Added pre-release header as well.

charlieconcur commented:
:shipit:
